notify

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This file is public domain in the USA and all free countries.
+# Elsewhere, consider it to be WTFPLv2. (wtfpl.net/txt/copying)
+
+# check if $1 is a number
+re='^[0-9]+$'
+if [[ $1 =~ $re ]] ; then
+	SLEEP="$1"
+else
+	SLEEP=10 # time between time notifications
+fi
+
+# output current time every $1 seconds
+while sleep $SLEEP
+do
+	date "+* It's %k:%M:%S o' clock ..."
+done
+
```

Comment:
This is an example of a dumb background job posting the current time every XX seconds to a chat.
It's used as an example backgound job in commands.sh